setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+setup(
+  name = 'config_maker',
+  packages = ['config_maker'],
+  scripts=['bin/make_config'],
+  version = '0.0.1',
+  description = 'Create config files from environment',
```

Comment:
typo: CreateS
or maybe it is better to say:  Builds config file collecting data from environment variables